### PR TITLE
Fix: add missing import for checkbox selection

### PR DESCRIPTION
**Pull request by Poulinhoo**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Fixing the problem on the new carrier creation page the select/unselect all options for group access
| Type?             | bug fix
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | Go to the new carrier page and create one, use the select all checkbox, it should select all options.

without fix

https://github.com/user-attachments/assets/4c4456da-d017-4a3a-b532-62e44c5672e8

with fix

https://github.com/user-attachments/assets/831ef985-30c8-4bca-ae8d-9a56478bfe91

### CODE REVIEW

**Comment by ps-jarvis** [comment]
Hi, thanks for this contribution!

Please consider opening an issue before submitting a Pull Request:
* If it's a bug fix, it helps maintainers verify that the bug is effectively due to a defect in the code, and that it hasn't been fixed already.
* It can help trigger a discussion about the best implementation path before a single line of code is written.
* It may lead the Core Product team to mark that issue as a priority, further attracting the maintainers' attention.

(Note: this is an automated message, but answering it will reach a real human)

**Comment by ps-jarvis** [comment]
Hello @PoulainMaxime!

This is your first pull request on [PrestaShop](https://github.com/PrestaShop/PrestaShop/) repository of the PrestaShop project.

Thank you, and welcome to this Open Source community!

**Review by jolelievre — APPROVED** [review]

**Comment by Robin-Fischer-PS** [comment]
Hello,
The PR can be merged once approved by another dev, QA ✔️

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Thank you @PoulainMaxime

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
